CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Next release
 
+...
+
+## Mypy 1.14 (unreleased)
+
+We’ve just uploaded mypy 1.14 to the Python Package Index ([PyPI](https://pypi.org/project/mypy/)). Mypy is a static type
+checker for Python. This release includes new features, performance improvements and bug fixes.
```

Comment:
Are there perfromance improvements? I didn't see any from a quick read-through, but usually these are grouped into their own section if so.

Reply:
Ops. It's a copy-paste blurb. I guess there aren't any specific perf improvements, so we probably shouldn't advertize this.